fastjmd95/__init__.py

```diff
@@ -1 +1,2 @@
 from .jmd95numba import rho, drhodt, drhods
+from .jmd95wrapper import rho_x, drhodt_x, drhods_x
```

Comment:
```suggestion
from .jmd95wrapper import rho, drhodt, drhods
```

Let's think about how we would like to use this package. I would like to be able to write
```python
from fastjmd95 import rho
```
and have it just work.

That suggests we should
- Use the same names in the `jmd95wrapper` module
- Don't import from `jmd95numba` at all in `__init__.py`